base.ts

```
@@ -0,0 +1,307 @@
+    this.controller.abort("Closing LucraClient");
+    this.iframe?.remove();
+    this.iframe = undefined;
+  }
+
+  protected _sendMessage(message: any) {
+    try {
+      this.iframe?.contentWindow?.postMessage(message, this.iframeUrlOrigin());
+    } catch (e) {
+      console.error("Unable to send message to LucraClient iframe", e);
+    } finally {
+      return this;
```

Biome (2.4.9)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)